ppb/camera.py

```diff
@@ -96,10 +96,18 @@ def in_frame(self, sprite: BaseSprite) -> bool:
                 self.frame_bottom >= sprite.top
                 )
 
-    def translate_to_frame(self, point:Vector) -> Vector:
+    def translate_to_frame(self, point: Vector) -> Vector:
+        """
+        Converts a vector from PyGame to in-game coordinate space
+        """
         offset = (point - self.viewport_offset) * (1/self.pixel_ratio)
-        return self.position + offset
+        loc = self.position + offset
+        return loc.update(y=-loc.y)
 
-    def translate_to_viewport(self, point:Vector) -> Vector:
+    def translate_to_viewport(self, point: Vector) -> Vector:
+        """
+        Converts a vector from in-game to PyGame coordinate space
```

Comment:
This is more about screen space, not pygame specific (We'll have  a similar pixel array context no matter which rendering library we use.)

Reply:
Umm... You don't have too? Taking to discord.